apps/aecore/src/aec_governance.erl

```diff
@@ -33,6 +34,11 @@
 -define(TIMESTAMP_MEDIAN_BLOCKS, 11).
 -define(EXPECTED_BLOCK_MINE_RATE, 3 * 60 * 1000). %% 60secs * 1000ms * 3 = 180000msecs
 -define(BLOCK_MINE_REWARD, 10000000000000000000).
+%% Ethereum's gas limit is 8 000 000 and block time ~15s.
+%% For 3s block time it's 1 600 000 (5x less).
+-define(BLOCK_GAS_LIMIT, 1600000).
+%% Taken from Ethereum - a simple tx to send Eth is about 21000 gas.
+-define(TX_GAS, 21000).
```

Comment:
Should we not have a gas price depending on what it is we do? Now every tx seems equally expensive. I would have expected the gas for a contract call to be different from the gas of a name registration? Probably I miss something, but the comment: "Taken from Etherium" does not explain this constant well enough.

Reply:
This PR is just to enforce gas limit per micro block. It's a preparation for the following PRs that will implement variable gas for different txs.